libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.html

```diff
@@ -3,14 +3,12 @@
   [(ngModel)]="selectedOptions"
   (onChange)="onChange(selectedOptions)"
   [placeholder]="placeholder"
-  [maxSelectedLabels]="2"
-  selectedItemsLabel="{0} items selected"
   [overlayOptions]="overlayOptions"
   [filter]="filter"
   (onFilter)="onSearch($event)"
   [virtualScroll]="true"
-  [virtualScrollItemSize]="50"
-  [virtualScrollOptions]="scrollOptions"
+  [virtualScrollItemSize]="itemHeight"
```

Comment:
Another potential confusion around size. Use "height" instead of "size" here as it's more descriptive.

Reply:
See comment above - the `virtualScrollItemSize` is default setting name from the component. I agree it's confusion, so I change the input name to `itemHeight`.